themes/digital.gov/src/scss/new/_gh-commit-info.scss

```diff
@@ -0,0 +1,31 @@
+@use "uswds-core" as *;
+
+.gh-commit-info {
+  margin: units("105") 0;
+
+  &__container {
+    align-items: center;
+    border-top: 1px color("base-light") solid;
+    display: flex;
+    justify-content: center;
+    margin-top: units(2);
+    padding: units(2) 0;
+  }
+
+  &__edit-button {
+    @include u-font("sans", "xs");
+    background-color: white;
+    border: 2px color("blue-warm-50v") solid;
+    border-radius: 0.25rem;
+    color: color("blue-warm-50v");
+    font-weight: 600;
+    padding: units("2px") units(1);
+    text-decoration: none;
+  }
```

Comment:
We can get rid of these styles if we use the button outline variant.

The classes are: `.usa-button.usa-button--outline`.

Reply:
Good call, removed styles and added `.usa-button.usa-button--outline`